app/(site)/temp/all-games/AllGamesDisplay.tsx

```diff
@@ -20,65 +20,55 @@ export default function AllGamesDisplay({
+                                    src={game.image}
+                                    alt={`${game.title} cover`}
+                                    className="w-full h-full object-cover"
+                                    width={1000}
+                                    height={1000}
+                                    quality={100}
+                                />
+                            ) : (
+                                <div className="flex items-center justify-center w-full h-full bg-gray-700">
+                                    <span className="text-gray-400">No Image</span>
+                                </div>
+                            )}
+                        </figure>
 
-                    return (
-                        <div
-                            key={game.gid}
-                            className="flex flex-col md:flex-row items-center bg-gray-800 rounded-lg p-4 shadow-lg"
-                        >
-                            {/* Game Image */}
-                            <div
-                                className="flex-shrink-0 w-60 h-56 overflow-hidden rounded-lg bg-gray-700"
-                                style={{ flexBasis: "22rem" }}
+                        {/* Game Content */}
+                        <div className="card-body bg-neutral">
+                            <Link
+                                href={`/game/${game.gid}`}
+                                className="card-title hover:text-secondary transition-colors duration-300"
                             >
-                                {game.image ? (
-                                    <Image
-                                        src={game.image}
-                                        alt={`${game.title} cover`}
-                                        className="w-full h-full object-contain"
-                                        width={1000}
-                                        height={1000}
-                                        quality={100}
-                                    />
-                                ) : (
-                                    <div className="w-full h-full flex items-center justify-center">
-                                        <span className="text-gray-400">No Image</span>
-                                    </div>
-                                )}
-                            </div>
-
-                            {/* Game Details */}
-                            <div className="ml-4 flex-grow">
-                                <h2 className="text-xl font-bold">{game.title}</h2>
-                                <p className="text-gray-400">Developer: {game.developer}</p>
-                                <div className="mt-2 text-sm">
-                                    <span>${game.price}</span>
-                                </div>
-                                {/* <div className="mt-1 text-xs text-gray-500">
```

Comment:
Same thing here: "I would not remove this, we might need it for later. We should only remove at the end."

Reply:
added back